modules/ui.styl

```diff
@@ -200,10 +201,18 @@ formElementBase() // This could be a class that is extended rather than a mixin
       color: var(--color-black-soft)
       cursor: not-allowed
 
+  &--small-text
+    font-weight: 500
+    font-size: 1rem
+
+
 .ui-input
   formElementBase()
   width: 100%
 
+  &--black
```

Comment:
Again need to ensure that this style is based on the Figma designs. Not sure of this alt state for form elements. If so then the descriptive varient class should be `ui-input--border-black` or similar. Just as black it could be read that the primary color of the input was black

Reply:
There isn't any figma designs for this yet but will bring it up in review

Reply:
I need to detangle the donation site and the main site in terms of this library, otherwise the committed page and new pages are waiting on the donation site being finished. Is there a way that I can come back to things like this in retrospect and focus on the library - main site relationship?

Reply:
Refactored